### ISSUE TITLE
Gallery webpage needs slight update

### ISSUE THREAD

**Issue by C13L0** [phsc-edu/2018#411]
From: Maria Rhew
Sent: Wednesday, August 29, 2018 11:55 AM
To: Webmaster
Subject: Gallery webpage needs slight update
 
Hi Cielo,

Here is the link for the page that needs to be updated:

https://phsc.edu/student-life/rao-musunuru-md-art-gallery

This is the content that should be removed:

In its final days...
Ninth Annual Faculty & Staff Summer Art Show
Join us on a fascinating, creative journey with the talented PHSC faculty and staff as they display their creative best in show.   ~closes Aug. 9, 2018

Thank you so much for helping out with this, you are the best!

Don't forget to come and visit the dinos -- they are amazing!!!

^_^

Maria  
Maria Rhew

Staff Assistant, West Campus Library  

Pasco-Hernando State College

10230 Ridge Road

New Port Richey, FL 34654  

727.816.3231

rhewm@phsc.edu

**Comment by Computerrat** [comment]
please review

**Comment by curtiskphsc** [comment]
@Computerrat This was completed yesterday. I deleted the block of text as requested. Did I miss something?
![gallery](https://user-images.githubusercontent.com/28155245/44858458-11b8ac80-ac40-11e8-9776-dc73f9b9aac7.PNG)

**Comment by C13L0** [comment]
@curtiskphsc @boettcc 

When editing content, please make sure and check for accessibility. Please correct the issue in your screenshot where a title is typed in all caps.

**Comment by C13L0** [comment]
This has been completed

> 08/30/2018 - 16:44 by curtisk
> corrected title

> 08/29/2018 - 16:03 by curtisk
> Removed faculty/staff show info per Maria Rhew.
